customer/src/test/java/no/unit/nva/customer/create/CreateCustomerHandlerTest.java

```diff
@@ -148,6 +149,20 @@ void shouldReturnConflictErrorWhenTryingToCreateCustomerForInstitutionThatIsAlea
         assertThat(response.getStatusCode(), is(equalTo(HttpURLConnection.HTTP_CONFLICT)));
     }
 
+    @Test
+    void shouldReturnPublicationTypesWhenValueIsSet() throws BadRequestException, IOException {
+        var randomPublicationTypes = randomPublicationTypes();
+        var customerDto =
+            CustomerDto.builder()
+                .withName("New Customer")
+                .withPublicationTypes(randomPublicationTypes)
+                .build();
+        var requestBody = CreateCustomerRequest.fromCustomerDto(customerDto);
+        var response = executeRequest(requestBody, CustomerDto.class);
+        var actualResponseBody = CustomerDto.fromJson(response.getBody());
+        assertThat(actualResponseBody.getPublicationTypes(), is(equalTo(randomPublicationTypes)));
+    }
+
```

Comment:
Can you add a test to ensure that the publication response actually serializes to the expected format? Since we are applying data from Publication, these values need to be in the same namespace since we will at some point want to compare the values. It's useful for users to be able to compare real values (i.e. URIs).